server/server.js

```diff
@@ -1,15 +1,152 @@
 const express = require("express");
+const cors = require("cors");
 const app = express();
 const port = process.env.PORT || 5000;
+const { Pool } = require("pg");
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+// Create a PostgreSQL connection pool using the DATABASE_URL environment variable
+const db = new Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: false,
+});
+
+app.use(express.json());
+app.use(cors());
+
+function generateUniqueId() {
+  return Math.floor(Math.random() * 1000000) + 1;
+}
+
+// Error handling middleware
+app.use((err, req, res, next) => {
+  console.error("Error:", err);
+  res.status(500).json({
+    result: "failure",
```

Comment:
Oh well this is cool! Didnt know you could do that

Reply:
Thank you! Google helped me to know about it :D